src/components/stepper/stepper/stepper.e2e.ts

```diff
@@ -242,4 +282,88 @@ describe('sbb-stepper', () => {
   });
+
+  it('focuses the correct element in the step cntent', async () => {
+    const stepLabelOne = element.querySelector<SbbStepLabelElement>(
+      'sbb-step-label:nth-of-type(1)',
+    )!;
+
+    await sendKeys({ down: 'Tab' });
+    expect(document.activeElement!.id).to.be.equal(stepLabelOne.id);
+
+    await sendKeys({ down: 'Tab' });
+    expect(document.activeElement!.id).to.be.equal('next-button-1');
+  });
+
+  it('selects tab on right arrow key pressed', async () => {
+    const stepLabelTwo = element.querySelector<SbbStepLabelElement>(
+      'sbb-step-label:nth-of-type(2)',
+    )!;
+
+    await sendKeys({ down: 'Tab' });
+    await sendKeys({ down: 'ArrowRight' });
+
+    expect(stepLabelTwo).to.have.attribute('data-selected');
+    expect(stepLabelTwo.step).to.have.attribute('data-selected');
+  });
+
+  it('selects tab on left arrow key pressed', async () => {
+    const stepLabelOne = element.querySelector<SbbStepLabelElement>(
+      'sbb-step-label:nth-of-type(1)',
+    )!;
+    const stepLabelTwo = element.querySelector<SbbStepLabelElement>(
+      'sbb-step-label:nth-of-type(2)',
+    )!;
+
+    await sendKeys({ down: 'Tab' });
+    await sendKeys({ down: 'ArrowRight' });
+
+    expect(stepLabelTwo).to.have.attribute('data-selected');
+    expect(stepLabelTwo.step).to.have.attribute('data-selected');
+
+    await sendKeys({ down: 'ArrowLeft' });
+
+    expect(stepLabelOne).to.have.attribute('data-selected');
+    expect(stepLabelOne.step).to.have.attribute('data-selected');
+  });
+
+  it('wraps around on arrow key navigation', async () => {
+    const stepLabelOne = element.querySelector<SbbStepLabelElement>(
+      'sbb-step-label:nth-of-type(1)',
+    )!;
+    const stepLabelTwo = element.querySelector<SbbStepLabelElement>(
+      'sbb-step-label:nth-of-type(2)',
+    )!;
+    const stepLabelThree = element.querySelector<SbbStepLabelElement>(
+      'sbb-step-label:nth-of-type(3)',
+    )!;
+
+    await sendKeys({ down: 'Tab' });
+    await sendKeys({ down: 'ArrowRight' });
+
+    expect(stepLabelTwo).to.have.attribute('data-selected');
+    expect(stepLabelTwo.step).to.have.attribute('data-selected');
+
+    await sendKeys({ down: 'ArrowRight' });
+
+    expect(stepLabelThree).to.have.attribute('data-selected');
+    expect(stepLabelThree.step).to.have.attribute('data-selected');
+
+    await sendKeys({ down: 'ArrowRight' });
+
+    expect(stepLabelOne).to.have.attribute('data-selected');
+    expect(stepLabelOne.step).to.have.attribute('data-selected');
+  });
+
+  it('wraps around on arrow left arrow key navigation', async () => {
+    const stepLabelThree = element.querySelector<SbbStepLabelElement>(
+      'sbb-step-label:nth-of-type(3)',
+    )!;
+
+    await sendKeys({ down: 'Tab' });
+    await sendKeys({ down: 'ArrowLeft' });
+
+    expect(stepLabelThree).to.have.attribute('data-selected');
+    expect(stepLabelThree.step).to.have.attribute('data-selected');
+  });
```

Comment:
Nice tests which you added. thank you!

Reply:
Last test question, should we also test / assert the labelled by link from step to step-label?